doc/zmq_setsockopt.txt

```diff
@@ -523,7 +523,7 @@ NOTE: in DRAFT state, not yet available in stable releases.
 
 [horizontal]
 Option value type:: char[2]
-Option value unit:: 0, MIN_ZMTP_VERSION_1_0, MIN_ZMTP_VERSION_2_0, MIN_ZMTP_VERSION_3_0, MIN_ZMTP_VERSION_3_1
+Option value unit:: 0, ZMQ_ZMTP_STRICT_V1_0, ZMQ_ZMTP_STRICT_V2_0, ZMQ_ZMTP_STRICT_V3_0, ZMQ_ZMTP_STRICT_V3_1
```

Comment:
What I meant with this was not just to rename the option, but to take away entirely the ability to specify a version - I really don't want to have to start debugging people's setting incompatible versions across nodes who then open bugs about it.
Just make this a boolean please, and if set to true, reject v1 which doesn't have a signature.

Reply:
I'm just going to remove the whole thing -- it's not critical for what I need to do, and is just complicating things.  This should probably have been its own PR in the first place, so that's on me.